src/sync.d

```diff
@@ -131,11 +131,12 @@ private Item makeItem(const ref JSONValue driveItem)
 		// https://github.com/abraunegg/onedrive/issues/11
 		if (isItemRemote(driveItem)) {
 			item.mtime = SysTime.fromISOExtString(driveItem["remoteItem"]["fileSystemInfo"]["lastModifiedDateTime"].str);
+			log.vdebug("Remote item time: ", item.mtime);
```

Comment:
remote item here is not 'local' or 'remote', but a OneDrive object that resides on another drive id which is not the account default - typically this is onedrive shared folder